controllers/constants/runtime-metrics.go

```diff
@@ -15,21 +15,23 @@ limitations under the License.
 
 package constants
 
+// ${DASHBOARD_RATE_INTERVAL} is a placeholder for the metrics that needs to receive values from the dashboard at the
```

Comment:
This should be `${REQUEST_RATE_INTERVAL}`?

Reply:
actually this comment needs to be removed, thanks for the heads up.

Reply:
updated, thanks